packages/amplify-provider-awscloudformation/lib/upload-appsync-files.js

```diff
@@ -79,7 +94,7 @@ function uploadAppSyncFiles(context, resources) {
 function uploadAppSyncFile(context, fileName, filePath, s3LocationMap, buildTimeStamp) {
```

Comment:
Don't need this anymore I guess?

Reply:
This is still used to upload the schema file.